src/ssh_audit/hostkeytest.py

```diff
@@ -157,6 +157,7 @@ def perform_test(out: 'OutputBuffer', s: 'SSH_Socket', server_kex: 'SSH2_Kex', k
                 ca_key_type = kex_group.get_ca_type()
                 ca_modulus_size = kex_group.get_ca_size()
                 out.d("Hostkey type: [%s]; hostkey size: %u; CA type: [%s]; CA modulus size: %u" % (host_key_type, hostkey_modulus_size, ca_key_type, ca_modulus_size), write_now=True)
+                out.d("Raw hostkey bytes (%d): [%s]" % (len(raw_hostkey_bytes), raw_hostkey_bytes.hex()), write_now=True)
```

Comment:
Elsewhere, this repo uses `binascii.hexlify`, but since the built-in [`bytes.hex` has existed since 3.5](https://docs.python.org/3/library/stdtypes.html#bytes.hex), I figure it's fine to switch to that here.